src/ElectricPokemon.java

```diff
@@ -0,0 +1,24 @@
+public class ElectricPokemon extends Pokemon{
+    String food;
+    String quality;
+
+        public ElectricPokemon(String name, int xp, String yell){
+            super(name, xp, yell);
+            super.setType("Electric");
+        }
+
+    @Override
+    public void chargingSound(){
+        System.out.println(super.getName() + " is buzzing...");
```

Comment:
Deze override is eigenlijk in elke subklasse hetzelfde. JE had dit ook kunnen oplossen door "buzzing"  (in dit geval) als parameter mee te geven aan de methode. Misschien dan nog overschrijven door `super.chargingSound("buzzing")` te doen en hier een chargingSound() methode zonder parameter te definieren